Samples/Swift/DaysUntilBirthday/Shared/Services/UserProfileImageLoader.swift

```diff
@@ -22,10 +22,10 @@ typealias GIDImage = NSImage
 
 import Combine
 import SwiftUI
-import GoogleSignIn
+@preconcurrency import GoogleSignIn
 
 /// An observable class for loading the current user's profile image.
-final class UserProfileImageLoader: ObservableObject {
+@MainActor final class UserProfileImageLoader: ObservableObject {
   private let userProfile: GIDProfileData
   private let imageLoaderQueue = DispatchQueue(label: "com.google.days-until-birthday")
```

Comment:
Please change this to store the Task you created on line 42, and cancel the task in the deinit.

```suggestion
  private var imageLoadingTask: Task<Void, Never>?
```

And then:
```
    // In the init, assign the task to the property
    imageLoadingTask = Task {
        await loadProfileImage()
    }  
    // Cancel in the deinit
    deinit {
        imageLoadingTask?.cancel() // Cancel the task on deinit
    }

Reply:
Thanks for quick response. Done the changes as suggested.